deliverable-2/BUG-ANALYSIS.md

```diff
@@ -29,12 +30,12 @@ NOTE: We may have messed up this bug because we forked into our own repo by acci
 * Issue description:
 
 
-    When media content is playing in the browser, saying “mute this page” will mute the media. However, saying “unmute this page” wouldn’t unmute successfully. Furthermore, when a users says “mute”, the browser will mute the video that is playing for the user but when a user says “unmute”, the browser will not unmute the video for  the user
+    When media content is playing in the browser, saying “mute this page” will mute the media. However, saying “unmute this page” wouldn’t unmute successfully. Furthermore, when a user says “mute”, the browser will mute the video which is playing for the user but when a user says “unmute”, the browser will not unmute the video, instead it will search the word “unmute” for the user.
 
 * Work Estimation:
 
 
-    We first reproduced the bug. With our team’s knowledge of the architecture from last deliverable, we quickly identified where the bug appears in the project. We estimate that it’s a simple fix with no foreseeable complications during the implementation. Our estimation is that it would take less than one hour.
+    We first reproduced the bug. With our team’s knowledge of the architecture from last deliverable, we were able to identified a general direction of where the bug might appear in the project relatively quickly. We estimate that it should not be a difficult fix as such there is no foreseeable complications during the implementation. Our estimation is that it would take roughly an hour to finish implmentation and testing.
```

Comment:
Some suggestions:

1. ..., we were able to identify the source of this particular bug relatively quickly. (in the first sentence)
2. After we took some time to investigate the bug in the repository, we came to the conclusion that the bug will not be too difficult for us as the bug affects only a small amount of other components. Our estimation is that it would take roughly an hour to complete the implementation and verification for this bug.